src/olav_espenes_ex/ex05/myrand.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+
+__author__ = 'Olav Espenes'
+__email__ = 'olaves@nmbu.no'
+
+
+class LCGRand:
+    def __init__(self, seed):
+        self.seed = seed
+        self.a = 7**5
+        self.m = 2 ** 31 - 1
+
+    def rand(self):
+        while True:
+            self.seed = (self.a * self.seed) % self.m
+            return self.seed
+
+    def random_sequence(self, length):
+        return RandIter(self, length)
+
+    def infinite_random_sequence(self):
+        return RandIter(self, -1)
```

Comment:
This is not what we expected, but a smart way to do that.
We expected something like
```
while True:
   yield self.rand()
```
Ask the docstring specifies to use `yield` not `return`